### PR TITLE
Animations: Dev Portal & About

### PR DESCRIPTION
**Pull request by wilwade**
# Description

- Adding the animation to the dev portal and the about section
- Also has some tiny fixes for lg and sm

Part of #34 

## Screenshot(s)

About
image

Dev Portal
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,6 +46,7 @@
   },
   "type": "module",
   "dependencies": {
-    "@frequency-chain/style-guide": "^0.1.21"
+    "@frequency-chain/style-guide": "^0.1.21",
+    "@lottiefiles/dotlottie-web": "^0.37.0-beta.8"
```
Comment (wilwade) [inline package.json:50]: Beta works better.

**Comment by wilwade** [comment]
Oh. Now I see. I'll fix it in another pr

**Review by mattheworris — APPROVED** [review]
- Viewed preview on Chrome in multiple sizes, animations look good.
🚢 it!

**Comment by jeanettedepatie** [comment]
Also seeing this mistake: 
screenshota

**Comment by jeanettedepatie** [comment]
Is there supposed to be animation in the center dark blue bar that says Your:?

**Comment by jeanettedepatie** [comment]
The learn more about developer portal button does not work for me on Chrome Version 130.0.6723.117 (Official Build) (arm64) on a M2 Macbook Air.

**Comment by jeanettedepatie** [comment]
In the developer advantages section, under 3 Economical, the words "and affordable" should be removed from the end of the sentence.

**Comment by jeanettedepatie** [comment]
In the home page developer advantages section under 2, The copy should read: "   Build your app with open-source tools designed for a more decentralized environment free from reliance on big tech.  "

**Comment by jeanettedepatie** [comment]
When I submit a form, I get the thank you form, but then it takes me back to the old version of frequency.  I assume this will be fixed when we actually publish at the correct URL.

**Comment by jeanettedepatie** [comment]
Explore & Build.  Second sentence should read "  Frequency Developers Gateway offers a suite of self-hosted tools and services  ..."

**Comment by jeanettedepatie** [comment]
Of course we need to change Gateway to Frequency Developers Gateway throughout.

**Comment by jeanettedepatie** [comment]
I'm just going to highlight the stuff that needs to be implemented in yellow in the doc here:
https://docs.google.com/document/d/1BF5tmXFm-pZu7pQS5-Hipw4du2lvJzecEf2BA3HrTKA/edit?tab=t.0

**Comment by jeanettedepatie** [comment]
Animations look good though.

**Pull request merged**